dask_cuda/tests/test_spill.py

```diff
@@ -207,57 +206,58 @@ async def test_cupy_cluster_device_spill(params):
 async def test_cudf_cluster_device_spill(params):
     cudf = pytest.importorskip("cudf")
 
-    # Disabling compression via environment variable seems to be the only way
-    # respected here. It is necessary to ensure spilled size matches the actual
-    # data size.
-    with patch.dict(os.environ, {"DASK_DISTRIBUTED__COMM__COMPRESSION": "False"}):
```

Comment:
The good news is that the config approach seems to work fine. The bad news is that this `env` approach does not :/

Not yet sure if there is a clean way to support both.